releasenotes/notes/fix-staged-passmanager-c2e423ebf17eb57e.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Handles repeated stage names provided on :class:`qiskit.transpiler.StagedPassManager`
+    initialization. UserWarning is thrown, and only first occurances are preserved.
```

Comment:
I'd argue this should maybe be an upgrade note since it's a behavior change when switching between the old release. Since, instead of having duplicated stages run more than once this now only preserves the first occurrence. Which is something we'd need to document for existing users of `StagedPassManager`.